src/sqlfluff/utils/analysis/select_crawler.py

```diff
@@ -145,13 +145,9 @@ def as_json(self) -> Dict:
         if self.query_type != QueryType.Simple:
             result["query_type"] = self.query_type.name
         if self.selectables:
-            result["selectables"] = [
-                s.as_str() for s in self.selectables
-            ]  # type: ignore
+            result["selectables"] = [s.as_str() for s in self.selectables]
```

Comment:
3.11 thinks that this ignore is not needed, but someone with an old mypy would get the opposite error when they try to commit.

Reply:
what's the error? There might be another way around the type hinting here to make both of them happy.

Reply:
actually - I'm running an older version so I can check myself! Have just pushed a fix which should work for both.